pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java

```diff
@@ -1188,7 +1188,8 @@ public ResultSet getProcedures(@Nullable String catalog, @Nullable String schema
       @Nullable String procedureNamePattern)
       throws SQLException {
     String sql;
-    sql = "SELECT NULL AS \"PROCEDURE_CAT\", n.nspname AS \"PROCEDURE_SCHEM\", p.proname AS \"PROCEDURE_NAME\", "
+    sql = "SELECT current_database() AS \"PROCEDURE_CAT\", "
+          + " n.nspname AS \"PROCEDURE_SCHEM\", p.proname AS \"PROCEDURE_NAME\", "
           + "NULL, NULL, NULL, d.description AS \"REMARKS\", "
           + DatabaseMetaData.procedureReturnsResult + " AS \"PROCEDURE_TYPE\", "
           + " p.proname || '_' || p.oid AS \"SPECIFIC_NAME\" "
```

Comment:
I hadn't seen this until now. I would think one should be able to use SPECIFIC_NAME to refer to an object in the database, no? This would create a name such as 123_somefunction which while unique can't be used to find the function.

Reply:
@davecramer I did not add or update line 1194/1195. This is existing code.

Reply:
you are correct. thanks